### PR TITLE
Choco coins are now premium items

### PR DESCRIPTION
**Pull request by SonixApache**
This way you can trade real coins for edible coins!
I did this because I thought it would be funny and cute.

🆑 
 - tweak: Choco-coins are now premium (coin-only) items.

### CODE REVIEW

**Comment by MadmanMartian** [comment]
incoming greytide

**Comment by Irradiation** [comment]
You did this because I dared you and you actually wanted to spite a discount slav.

**Comment by boy2mantwicethefam** [comment]
thats it im getting me mallet

**Comment by aceedex** [comment]
I see your true intentions c*der man

**Comment by yclatious** [comment]
You have offended the greytide today. This is going on the book of slights, you fucking pseudo belgian.

**Comment by ShiftyRail** [comment]
I see no real reason for it
The chococoin is just a practical way to get premium items when there is no miner and/or before the miners arrive (ie, to rush those insulated gloves).
It's a useful usage and it serves a purpose in the greytide ecosystem

**Comment by jknpj** [comment]
Space ain't free. Security is going to :-1: because it messes with their malf kit powergaming and greyshirts because it messes their sparerushing. Maint has coins, it should be :ok_hand: .

**Comment by yclatious** [comment]
You are also in the fucking book now jknpj. The tide cometh, you am*rican.

**Comment by Irradiation** [comment]
Nuclear perspective: what about people with no maint access? There's 1 or 2 coins in public areas at best on "most" maps which will probably get nabbed by the coin rushers.

**Comment by Blithering** [comment]
only if you replace the regular vending machine slot with chocolate insulated gloves

**Comment by SonixApache** [comment]
@ShiftyRail  @Irradiation
QM has two coins in his office on every map that you could trade for / steal, though on some maps they're in a locker for some reason. Should probably equalize that across the board.
There's at least one coin mint mapped in on every map, and they're not hard to build. They're also not access-locked.

Also I don't believe in a 'greytide eco system' -- let them put in some effort if they're so desperate?

**Comment by jknpj** [comment]
The 'greytide ecosystem' gloves come mainly from cargo anyways

**Comment by yclatious** [comment]
The way I see it, coins are one of the barriers which prevent the tide from going even worse when looking for gloves. There are indeed several several ways to get glubbs but I really gotta ask if you wanna cull the ways to get em and then encourage greyshits to break into places to then bust open lockers and whatnot, because that will just increase the shittery done for the sake of the yellow plastic.

I went and did major break and entry and then crafted about 9 ieds just to bust open a electrical crate and even if I didnt have that oportunity (this was packed), then Id probably just go and weld down walls until I found gloves inside the place. If a greyshit actually goes to cargo to get gloves, is he even a greyshit? Nah, that shits weak. 

You might ask why I did all that just to get a pair of gloves when 20 credits and a law 2 to a borg could have gotten it quicker. Simple. Because without the effort, without either a rush, a competition or some kind of obstacle, whats the point? Gloves give one greyshit the ability to grey as much as he likes and to share his knowledge, which while breaking what I just aid, is something that allows a good chunk of freedom and opens up possibilities, the rest scavenge or learn were the spots are, but enabling one is usualy good as it encourages folks to talk to one another, cooperate in projects, help minimize the b&e when getting more glubbs. You might disagree, but all I say here is from my experiences as a greyshit, so eh.

tldr Leave my fucking chokycoins alone or I will break into engie every single round and go irish carbomb the electrical lockers and you folks know I will.

**Comment by SonixApache** [comment]
Why don't you just weld into the QM's office to steal the coins in there?

**Comment by yclatious** [comment]
Because Cargonia is holy grounds for our bros, you dont just tresspass into your authoritarian brothers porch, even if you are a anarcho capitalist.

Do you even greyshit dude.

**Comment by SonixApache** [comment]
why don't you just use the string coin to make more coins

**Comment by yclatious** [comment]
Because what weak ass kinda motherfucker double dips on fucking gloves or lets his fellow bros be so cuddled he straight up gives em the coin for it.

You say the wire, if they are polite, and wish em good luck, after that they scrounge for a multitool or wirecutter and pray you werent antag and told em the shock one so they can break into places and sneak out with a new pair of shiny yellow.

**Comment by SonixApache** [comment]
perhaps changing the ecoinomy will bring you a step closer to the land of the sane

**Comment by yclatious** [comment]
Perhaps nuking your country will purge your weak nigga ass from ever touching my coins or glubbs ever again.

**Comment by kilozombie** [comment]
We can talk about greytide but I've used choco coin purchases for premium items as literally every job
It's valuable to have a consistent way of getting coins since Getmores are easier to find than the one or two consistent spawns dotted throughout the maps in different places, and MUCH easier than the random spawns dotted throughout Maintenance.

t. barely plays the game right now though

**Comment by SonixApache** [comment]
okay, i guess this isn't cute/funny enough to cut it

**Comment by kilozombie** [comment]
But trains were hilarious

**Comment by yclatious** [comment]
Fission Mailed, we got em bois.

**Comment by MadmanMartian** [comment]
coward

**Comment by SonixApache** [comment]
fine, the collabs can close it in a week

**Comment by bathosbathtime** [comment]
Powerful and brave
I'm against this change but I appreciate you committing to the joke.

**Comment by DrSnips** [comment]
You think this is a joke? You're dead wrong. The day of the grey rope is soon.

**Comment by Kitfox88** [comment]
remove tide

**Comment by SonixApache** [comment]
What if I moved the coin mint to be on-station, located in cargo somewhere instead of on the roid?

**Comment by yclatious** [comment]
Cue greyshits imediately rushing cargo and either breaking in forcefully, fucking up the windows and airlocks, or it becoming a roundstart ritual of cargo handing out coins. Or the unholy mix of assistants breaking into cargo to mint metal they "borrowed" and giving everyone their own coin. 

Yamete, sonix-kun senpai.

**Comment by ghost** [comment]
If you want to give money value, this is a good place to start. Money has very little value because crew cannot directly exchange it for power. Coins on the other hand are stupid and should be rethought entirely, as they introduce a secondary currency that competes with actual credits.

**Comment by SonixApache** [comment]
There was an interesting discussion with @DrSnips about making coins have value based off their materials, and allowing coins to give discounts on other vending items / have premium premium items.

I'm not sure about secondary currencies being a good or bad thing. As-is, the entire economy (both creds and coins) collapses when cargo/mining puts in about a minute of effort.

**Comment by thatsmyhorse2** [comment]
So wait

You want to remove one of the only legitimate uses for civilian money in the game (an expensive way to get premium items)
Because ONE premium item is ONE of the ways ONE job (assistant) can hack doors

And after this change, guess which job is still top 3 easiest and fastest jobs to get a coin as to get their glubbs anyways? A-assistants...

This is the most shortsighted knee-jerk c*der nerf I have ever seen. If you are even capable of self-awareness you should be fucking embarrassed for doing this even ironically.

**Comment by Blithering** [comment]
honestly, greyshirts would just continue stealing the gloves themselves. There's only 2 machines that give gloves and one is in engineering anyway, so they might as well just take a detour into tech storage or telecomms. the real victims would be gimmick clowns/chaplains who need 3 coins to get a full costume set from their vendors.

**Comment by bathosbathtime** [comment]
The joke of "give a coin to get a coin" is one I enjoy but in terms of balance, it would probably go smoother if you just removed glubbs from the vendor.

**Comment by DrSnips** [comment]
Haha how is needing gloves even real haha? Just play mechanic and walk into the captain's office haha.

**Comment by SonixApache** [comment]
@thatsmyhorse2 I'm not sure what you're on about nor what you think my motivations are for making this, if you're even talking to me.

@Blithering the glubbs thing is pretty secondary, but there's plenty of ways to get glubbs and they're only part of standard greyshirt attire because traditions and possibly construction. Once you hack one vending machine and one door, you basically don't need them anymore unless you're actually building or powergayming.
What did you mean by chocolate gloves anyway?

@bathosbathtime why not both? 🙂 (disclaimer: I see literally 0 reason to remove glubbs from vending machines)

**Comment by Accipitermaris** [comment]
Because premium vendor items being locked behind coins and not the in-game currency is a curiousity, and because coin-minting is very much a closed-loop one-and-done feature, I don’t think we’re missing out on any high quality Interdepartmental Interaction(tm) or gameplay via the vending machine availability of Choco-Coins

(That isn’t to condemn coins conceptually, as a coin/poker chip-based station economy would probably be a far easier base for the coveted “Balanced Station Economy”, although I think the coders for whom that was a project are no longer with us.)

**Comment by Blithering** [comment]
@SonixApache the sticking point for this PR seems to be the greyshirt run to arrivals vendors, get chococoin, get glubbs from youtool ritual. I was joking that you should cut the middleman and have the vendor sell chocolate gloves instead. Also I've found if you spend a lot of time in maintenance, glubbs can help avoid the ever embarrassing case of clicking a wire and instahusking yourself.

**Comment by SonixApache** [comment]
Choco-glubbs from the getmore, gotcha. I'll consider it.
